eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml

```diff
@@ -21,4 +21,4 @@ steps:
           }
         }
       displayName: Prevent changes to eng/common outside of azure-sdk-tools repo
-      condition: and(succeeded(), ne(variables['Skip.EngCommonWorkflowEnforcer'], 'true'))
+      condition: and(succeeded(), ne(variables['Skip.EngCommonWorkflowEnforcer'], 'true'), not(endsWith(variables['Build.Repository.Name'], '-pr')))
```

Comment:
Why did you have to shift this to the runtime condition instead of the template condition? Is that because `Build.Repository.Name` isn't available at template time?

Reply:
Yes thats why.